tests/tests_signin_user.py

```diff
@@ -0,0 +1,7 @@
+def test_with_authenticated_client(client, django_user_model):
+    username = "test"
+    password = "test"
+    django_user_model.objects.create_user(username=username, password=password)
+    client.force_login(user)
```

Comment:
user is not defined.
On the line above, you may want to do:
`user = django_user_model.objects.create_user(username=username, password=password)`